packages/manager/apps/dedicated/client/app/dedicated/server/dashboard/advanced-features/sgx/introduction/introduction.html

```diff
@@ -0,0 +1,40 @@
+<oui-back-button on-click=":: $ctrl.goBack()"
+    ><span
+        data-translate="dedicated_server_dashboard_advanced_features_sgx_introduction_go_back"
+    ></span
+></oui-back-button>
+
+<h2
+    data-translate="dedicated_server_dashboard_advanced_features_sgx_introduction_header"
+></h2>
+
+<h3
+    data-translate="dedicated_server_dashboard_advanced_features_sgx_introduction_subheader"
+></h3>
+
+<p
+    data-translate="dedicated_server_dashboard_advanced_features_sgx_introduction_presentation"
+></p>
+
+<oui-button variant="primary" class="mb-3" on-click="$ctrl.goToManage()"
+    ><span
+        data-translate="dedicated_server_dashboard_advanced_features_sgx_introduction_manage_button"
+    ></span
+></oui-button>
+
+<div class="row" data-ng-if="$ctrl.tileDocumentationUrl">
+    <oui-tile
+        heading="{{:: 'dedicated_server_dashboard_advanced_features_sgx_introduction_manage_tile_header' | translate }}"
+        class="col-md-4"
+    >
+        <oui-tile-definition>
+            <oui-tile-description>
+                <a
+                    data-ng-href="{{:: $ctrl.tileDocumentationUrl }}"
+                    target="_blank"
+                    data-translate="dedicated_server_dashboard_advanced_features_sgx_introduction_manage_tile_content"
+                ></a>
```

Comment:
Nothing to specify it opens in a new tab ?

Reply:
Done !